Updated data sources link to point to datacommons.org/data

dwnoble opened the pull request:
* Removed links to https://docs.datacommons.org/datasets/ and replaced them with links to https://datacommons.org/data
* Updated legacy https://datacommons.org/datasets/ redirect to point to https://datacommons.org/data

gmechali reviewed (approved):
This link still seems broken: https://docs.datacommons.org/data

Other than that, very neat. I didnt know we had those data pages!

kmoscoe commented:
> This link still seems broken: https://docs.datacommons.org/data
> 
> Other than that, very neat. I didnt know we had those data pages!

There is no such URL. What are you trying to do here, Dan?

dwnoble commented:
> > * Removed links to https://docs.datacommons.org/datasets/ and replaced them with links to https://datacommons.org/data
> > * Updated legacy https://datacommons.org/datasets/ redirect to point to https://docs.datacommons.org/data
> 
> I think you just mean https://datacommons.org/data, right? (there is no docs.datacommons.org/data)

Good catch- that is right i meant datacommons.org/data. Updated the comment!

dwnoble commented:
> This link still seems broken: https://docs.datacommons.org/data
> 
> Other than that, very neat. I didnt know we had those data pages!

Fixed the link in the comment to point to datacommons.org/data
